app/src/test/java/com/kickstarter/libs/utils/DateTimeUtilsTest.kt

```diff
@@ -127,7 +119,7 @@ class DateTimeUtilsTest : KSRobolectricTestCase() {
             "Dec 17, 2015, 1:35:05 PM",
             DateTimeUtils.mediumDateTime(
                 DateTime.parse("2015-12-17T18:35:05Z"),
-                DateTimeZone.forID("EST")
```

Comment:
"EST" no longer recognized as ID, to assert for that time zone: 
DateTimeZone.forOffsetHours() if it’s always fixed UTC−5, the offset in hours to check for "EST"